### PR TITLE
Document camera entity in Fully Kiosk Browser

### PR DESCRIPTION
**Pull request by tronikos**
## Proposed change

Document camera entity in Fully Kiosk Browser

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/119483
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
  - Added camera integration to Fully Kiosk Browser in Home Assistant.
  - Introduced camera controls including motion detection and screenshot capabilities.

### CODE REVIEW

**Review by cgarwood — DISMISSED** [review]

File: source/_integrations/fully_kiosk.markdown [hunk 1]
```diff
@@ -64,6 +66,10 @@ The following controls are available:
 - Play and stop media files
 - Set device volume
 
+The following is available as cameras:
+
+- Camera (requires motion detection on)
```
Comment (c0ffeeca7) [inline source/_integrations/fully_kiosk.markdown]:
```suggestion
- Camera (the camera only works in Fully Kiosk if the **Motion detection** is set to **On**. 
```

File: source/_integrations/fully_kiosk.markdown [hunk 2]
```diff
@@ -64,6 +66,10 @@ The following controls are available:
 - Play and stop media files
 - Set device volume
 
+The following is available as cameras:
```
Comment (c0ffeeca7) [inline source/_integrations/fully_kiosk.markdown]: I'm not sure I understand this. what is meant by "available as cameras"?

Comment (tronikos) [reply source/_integrations/fully_kiosk.markdown]: Reworded it a bit.

File: source/_integrations/fully_kiosk.markdown [hunk 3]
```diff
@@ -67,6 +69,10 @@ The following controls are available:
 - Play and stop media files
 - Set device volume
 
+The following is available as camera entity:
+
+- Camera (the camera only works in Fully Kiosk if the **Motion detection** is set to **On**.
```
Comment (frenck) [inline source/_integrations/fully_kiosk.markdown]:
```suggestion
- Camera (the camera only works in Fully Kiosk if the **Motion detection** is set to **On**).
```

**Review by frenck — APPROVED** [review]
Thanks, @tronikos 👍 

../Frenck

**Pull request merged**